v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerSimpleIT/mysql-schema.sql

```diff
@@ -15,3 +15,28 @@ CREATE TABLE `Users` (
     `startDate` date,      -- To: startDate DATE
     PRIMARY KEY (`id`)
 );
+
```

Comment:
Why is there no index in source schema? I see it in session file

Reply:
Done.